keps/sig-apps/1847-autoremove-statefulset-pvcs/kep.yaml

```diff
@@ -22,12 +22,12 @@ approvers:
 
 stage: alpha
 
-latest-milestone: "v1.23"
+latest-milestone: "v1.25"
```

Comment:
You're missing https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-apps/1847.yaml updated for PRR review.

Reply:
I updated 1847.yaml with the beta tag, is that sufficient? Also is it okay to leave Wojtek on there?